_ml-commons-plugin/api.md

```diff
@@ -115,6 +115,118 @@ The API returns information on the model, the algorithm used, and the content fo
 }
 ```
 
+## Upload load a model
+
+Use the upload operation to upload a pre-existing or custom model and into a model index. ML commons splits the model into smaller chunks to enable persistance in the cache.
```

Comment:
```suggestion
Use the upload operation to upload a model. ML commons will split the model into smaller chunks and save into model index.
```

Reply:
"...and save them into the model index."